pkg/manifests/configfile.go

```diff
@@ -97,24 +112,28 @@ func NewConfigFile(path, workingDir string) (*ConfigFile, error) {
 	case result.PatchUpdated != nil && result.PatchUpdated.PatchFile == "":
 		return nil, errors.New("patchUpdated's patchFile cannot be empty")
 	}
-	return &result, nil
+	return result, nil
 }
 
 // -- entry points for using a config file to generate or update manifests
 
 func makeNoCommandsRunErr(field string, cf *ConfigFile) error {
-	relConfigPath, err := cf.RelativeConfigPath()
-	if err != nil {
-		return fmt.Errorf("config file not relative to working dir: %s", err)
-	}
-	return fmt.Errorf("no %s commands to run in %s", field, relConfigPath)
+	return fmt.Errorf("no %s commands to run in %s (from path %s)", field, cf.RelativeConfigPath(), cf.originalPath)
 }
 
 // RelativeConfigPath returns the path to the config file, relative to
 // the working directory. This is used in error messages and to
 // identify resources generated from the config file.
-func (cf *ConfigFile) RelativeConfigPath() (string, error) {
-	return filepath.Rel(cf.workingDir, cf.path)
+func (cf *ConfigFile) RelativeConfigPath() string {
+	return cf.relativeConfigPath
+}
+
+// ConfigVsOriginalPath shows the config file as it relates to the
+// repo-relative path, e.g., `staging/../.flux.yaml`
+func (cf *ConfigFile) ConfigVsOriginalPath() string {
```

Comment:
I would call this `ConfigFileSearchPath()` or something in those lines, because it's the path used to find the configfile from the base path.

Reply:
I'll have another think about it. It doesn't need to be a path, I just though that was a compact representation of the two pieces of information (the path under consideration, and where the config file is in relation to it).